### ISSUE TITLE
Important Context Menu Options are in "More Options" 

### ISSUE THREAD

**Issue by siiparch** [moudey/Shell#99]
As stated in the title, some context menu options end up inside "More Options," for example, inside the File Explorer, the option "Open in a new tab" or "Open as Administrator." I use this software on various computers and on some, the aforementioned options are immediately noticeable and not found inside "More Options." I don't know what causes this behavior. The same problem occurs in both the "Beta" version and the "Stable" version.

Does anyone know how to solve that? Or what cause the problem?

**Comment by moudey** [comment]
There is a command in the Static Items section that moves items whose title exceeds approximately 20 characters to the "More Options" submenu. You can modify the number to 30 to bypass this issue.

```
item(where=this.title.len > 20 tip=this.title parent=@id.more_options.title)
```

**Comment by siiparch** [comment]
Many thanks for the prompt reply!

**Comment by siiparch** [comment]
Sorry to reopen this problem again. I've updated Shell to the latest version, and while the proble of "Open in a new tab" is solved automatically, I still have the exact same problem with the "Open as Administrator". 
I've seen that the software, or at least the settings, were highly changed.
I've tried to find the exact same line you've wrote above, but i think that there is no more, or it's changed.

Any idea on how can i solve this now? 
Thanks in advance
